node/runtime/src/lib.rs

```diff
@@ -17,6 +17,7 @@
 //! The Substrate runtime. This can be compiled with ``#[no_std]`, ready for Wasm.
 
 #![cfg_attr(not(feature = "std"), no_std)]
+#![recursion_limit="256"]
```

Comment:
Can you please answer (prefferable leaving a comment), why it's increased (like what dependency requires it)?